src/EnergyPlus/HeatRecovery.hh

```diff
@@ -64,17 +63,6 @@ struct EnergyPlusData;
 
 namespace HeatRecovery {
 
-    // Using/Aliasing
```

Comment:
Alright, this file is vastly reworked.  I'm not sure how much will be obvious on GitHub's view, so if you are curious, I would suggest just popping open the develop version and this branch next to each other, and scroll through to see the new code in action.